src/components/Posts/AllPost/AllPost.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Hidden from '@material-ui/core/Hidden';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Chip from '@material-ui/core/Chip';
```

Comment:
will it be better to read if to import all the `@material-ui/core/...` using import `import {... , ...} from '@material-ui/core'`

Reply:
I agree that it will be easier to read with `import {.., ..} from '@material-ui/core'`. However, this way we can reduce the bundle size for our users. You can check out this [stackoverflow question](https://stackoverflow.com/questions/41418165/more-efficient-es6-imports) for more info.